contracts/script/upgrade/upgrade_mainnet_v1_v2/UpgradeMainnet_V1_V2_P1.s.sol

```diff
@@ -0,0 +1,275 @@
+    function _deployRelayRegistry(InitParams memory initParams) internal {
+        createdContracts.relayRegistryImpl = address(new EigenDARelayRegistry());
+        createdContracts.relayRegistry = address(
+            new TransparentUpgradeableProxy(
+                createdContracts.relayRegistryImpl,
+                createdContracts.proxyAdmin,
+                abi.encodeCall(EigenDARelayRegistry.initialize, (initParams.existing.daOpsMsig))
+            )
+        );
+    }
+
+    function _deployDisperserRegistry(InitParams memory initParams) internal {
+        createdContracts.disperserRegistryImpl = address(new EigenDADisperserRegistry());
+        createdContracts.disperserRegistry = address(
+            new TransparentUpgradeableProxy(
+                createdContracts.disperserRegistryImpl,
+                createdContracts.proxyAdmin,
+                abi.encodeCall(EigenDADisperserRegistry.initialize, (initParams.existing.daOpsMsig))
+            )
+        );
+    }
+
+    function _deployPaymentVault(InitParams memory initParams) internal {
+        createdContracts.paymentVaultImpl = address(new PaymentVault());
+        createdContracts.paymentVault = address(
+            new TransparentUpgradeableProxy(
+                createdContracts.paymentVaultImpl,
+                createdContracts.proxyAdmin,
+                abi.encodeCall(
+                    PaymentVault.initialize,
+                    (
+                        initParams.existing.daOpsMsig,
+                        initParams.paymentVault.minNumSymbols,
+                        initParams.paymentVault.pricePerSymbol,
+                        initParams.paymentVault.priceUpdateCooldown,
+                        initParams.paymentVault.globalSymbolsPerPeriod,
+                        initParams.paymentVault.reservationPeriodInterval,
+                        initParams.paymentVault.globalRatePeriodInterval
+                    )
+                )
+            )
+        );
+    }
+
+    function _deployEjectionManager(InitParams memory initParams) internal {
+        createdContracts.ejectionManagerImpl = address(
+            new EjectionManager(
+                IRegistryCoordinator(initParams.existing.registryCoordinator),
+                IStakeRegistry(initParams.existing.stakeRegistry)
+            )
+        );
+    }
+
+    function _deployRegistryCoordinator(InitParams memory initParams) internal {
+        createdContracts.registryCoordinatorImpl = address(
+            new RegistryCoordinator(
+                IServiceManager(initParams.existing.serviceManager),
+                IStakeRegistry(initParams.existing.stakeRegistry),
+                IBLSApkRegistry(initParams.existing.blsApkRegistry),
+                IIndexRegistry(initParams.existing.indexRegistry),
+                ISocketRegistry(initParams.existing.socketRegistry)
+            )
+        );
+    }
+
+    function _deployEigenDAServiceManager(InitParams memory initParams) internal {
```

Comment:
nit - would be more intuitive when reading to have function names specify whether contract is net new (i.e, proxy/implementation) or pre-existing (i.e, implementation)